File: curso-python/ex081.py
cont = 0
lista = []

while True:
    n = int(input("digite um valor: "))
    lista.append(n)
    
    cont += 1

    resp = ' '
    while resp not in 'SNsn':
        resp = input("quer continuar [S/N]: ").strip().upper()
        
    if resp == 'N':
        print("saindo do programa...")
        break

lista.sort(reverse=True)

print(f"você digitou {cont} elementos")
print(f"os valores em ordem decrescente são {lista}")
print("o valor 5 faz parte da lista" if 5 in lista else "o valor 5  não faz parte da lista")

File: javascript-exercicios/ex081.js
const prompt = require("prompt-sync")()
let cont = 0
let lista = []

while (true){
    
    let n = Number.parseInt(prompt("Digite um valor: "))
    lista.push(n)
    cont += 1

    let resp = ' ';
    while (resp != 'S' && resp != 'N'){
        resp = prompt("quer continuar[S/N]: ").trim().toUpperCase();
    }
    if (resp == "N"){
        console.log("saindo do programa... ")
        break;
    }
}

lista.sort((a,b)=> b - a);

console.log(`você digitou ${cont} elementos`)
console.log(`os valores em ordem decrescente são ${lista}`)

if (lista.includes(5)){
    console.log("o valor 5 faz parte da lista")
}else{
    console.log("o valor 5 não faz parte da lista")
}